devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -69,6 +84,16 @@ def updated() {
 	initialize()
 }
 
+def configure() {
+	if (zwaveInfo.mfr.equals("0086") && zwaveInfo.model.equals("007A")) {
+		def commands = []
+		// Tell sensor to send us battery information instead of USB power information
+		commands << encap(zwave.configurationV1.configurationSet(parameterNumber: 0x5E, scaledConfigurationValue: 1, size: 1))
+		commands << encap(zwave.wakeUpV1.wakeUpIntervalSet(seconds: 4 * 3600, nodeid: zwaveHubNodeId))
```

Comment:
I think 4 hours is the default we automatically configure wakeup devices to use.

Reply:
The old code that did that was probably just stale.

Reply:
Done.